packages/contracts/contracts/Passport.sol

```diff
@@ -17,7 +21,6 @@ contract Passport is ERC721, Ownable, AccessControlEnumerable {
   mapping(uint256 => bool) public sold;
   uint256 public price;
   uint256 public supply;
-  uint256[] public tokenIds;
```

Comment:
Looks like we weren't using this anywhere?

Reply:
this was used in an older version, safe to remove